src/poc/server/poc-server/src/controllers/users.ts

```diff
@@ -1,7 +1,18 @@
-import {v4 as uuidv4} from "uuid";
+import { v4 as uuidv4 } from "uuid";
+const { createSecretKey } = require('crypto');
+import dotenv from "dotenv";
 import { Request, Response } from "express";
+import haversine from 'haversine-distance';
+import * as jose from 'jose';
 import { io } from "../index";
 export const sessionIds: string[] = [];
+export const ITBLocation = {
+    latitude: 43.25898815315478,
+    longitude: -79.92086378832343
+};
+
+dotenv.config();
+const secretKey = createSecretKey(process.env.JWT_SECRET, 'utf-8');
```

Comment:
the JWT_SECRET will have to exist in a .env file, not sure if there's a better way to store the secret